www/src/components/GridHelper/GridHelper.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const GridHelper = () =>
```

Comment:
Added the grid helpers, which can now be enabled by adding a `show-grids` class to the `<body />`, in Dev Tools.